schemas/src/test/resources/test-profiles/profile-test-simple-anyof-allof.json

```diff
@@ -25,11 +25,6 @@
         },
         {
           "allOf": [
-            {
```

Comment:
Was this an intentional change?

Reply:
yes to stop the generator crashing due to multiple formattedas constraints